MdePkg/Test/Mock/Include/GoogleTest/Ppi/MockReadOnlyVariable2.h

```diff
@@ -0,0 +1,47 @@
+/** @file MockReadOnlyVariable2.h
+  This file declares a mock of Read-only Variable Service2 PPI.
+  This ppi permits read-only access to the UEFI variable store during the PEI phase.
+  Copyright (c) Microsoft Corporation.
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+**/

```

Comment:
A little easier to read.

```suggestion
/** @file MockReadOnlyVariable2.h
  This file declares a mock of Read-only Variable Service2 PPI.
  
  This PPI permits read-only access to the UEFI variable store during the PEI phase.
  
  Copyright (c) Microsoft Corporation.
  SPDX-License-Identifier: BSD-2-Clause-Patent
**/

```